src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5698,16 +5658,14 @@ void DayltgGlare(EnergyPlusData &state,
 }
 
 void DayltgGlareWithIntWins(EnergyPlusData &state,
-                            Array1D<Real64> &GLINDX, // Glare index
-                            int const ZoneNum        // Zone number
+                            Array1D<Real64> &GLINDX,  // Glare index
+                            int const daylightCtrlNum // Current daylighting control number
```

Comment:
This is a little hard to comprehend. It is not the zone reference anymore it is the daylighting control num index, which I guess could imply a space, zone or enclosure.